### DISCUSSION TITLE
folder structure, album names, mv images to albums

### DISCUSSION THREAD

**Discussion by jrodan** [gilesknap/gphotos-sync#395] [Ideas]
Hey, 

i really wish to have the following features / options:

- one folder per year for photos
- album names in the following format: YYYY-mm-dd title
- mv images from photos to albums and do not show them in the main photo stream anymore (optionally: create symlinks back to photos)

Thanks!

**Comment by gilesknap** [comment]
Hi @jrodan thanks for the suggestion. I have a long standing intention to provide a generic way of having the user define their own structure using a folder and and filename template string. 

This is a moderately hard thing to do because of the name-clash logic.

Your idea has the extra requirement of making the albums first class. There are some complications here. How would you handle images that appear in two or more albums for example.

Unfortunately, because of the issues  with the Google API (see #119) I'm not putting a great deal of effort into this project and I don't think I'll ever get around to implementing any further folder structure changes.
